assets/js/translator.js

```diff
@@ -1252,6 +1249,52 @@ function autoSelectDstLang() {
     $('#dstLangSelect').val(curDstLang).change();
 }
 
+function setDefaultSrcLang() {
+    // Default for new users is first available Browser Preference Language pair
+    var browserLangs = window.navigator.languages; // Chrome, Mozilla and Safari
+    var ieLang = window.navigator.userlanguage || window.navigator.browserlanguage; // Internet Explorer
+    var prefSrcLang;
+    var i;
```

Comment:
This var can be inlined below.

Reply:
inlined below?

Reply:
it doesn't need to be declared separately, it can be inlined inside the loop

Reply:
Okay. :)